inmaputil/download.go

```diff
@@ -0,0 +1,105 @@
+package inmaputil
+
+import (
+	"github.com/mholt/archiver"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/cavaliercoder/grab"
+)
+
+func Download(path string) string {
```

Comment:
I don't think we want this function to be exported, so small d in download.

Reply:
Include comments above the definition of all functions that follow best practices: https://blog.golang.org/godoc-documenting-go-code